input/docs/getting-started/installation/index.md

```diff
@@ -79,10 +79,10 @@ ReactiveUI packages are now signed by the dotnet foundation. Only builds from th
 
 [Uno]: https://www.nuget.org/packages/ReactiveUI.Uno/
 [UnoBadge]: https://img.shields.io/nuget/v/ReactiveUI.Uno.svg
-[UnoDoc]: https://reactiveui.net/docs/getting-started/installation/uno-platform
+[UnoDoc]: https://platform.uno/blog/getting-started-with-uno-platform-and-reactiveui/
```

Comment:
Also updated Uno documentation links so they don't point to 404 but point to their blog posts.